tutorials/ab3TracingCode.md

```diff
@@ -6,21 +6,22 @@ pageNav: 3
 
 # {{ title }}
 
-
+<link rel="stylesheet" href="images/tracing/static/tracing.css">
 > Indeed, the ratio of time spent reading versus writing is well over 10 to 1. We are constantly reading old code as part of the effort to write new code. …[Therefore,] making it easy to read makes it easier to write.
 >
 > —  Robert C. Martin Clean Code: A Handbook of Agile Software Craftsmanship
 
 When trying to understand an unfamiliar code base, one common strategy used is to trace some representative execution path through the code base. One easy way to trace an execution path is to use a debugger to step through the code. In this tutorial, you will be using the IntelliJ IDEA’s debugger to trace the execution path of a specific user command.
 
 
-## Before we start
+<h2 id="before-we-start">Before we start</h2>
```

Comment:
No need. The anchor `before-we-start` is automatically generated by MarkBind. However, MarkBind will also warn about non-existent hash, which is a bug. Can ignore that for now.

Reply:
Noted